### PR TITLE
KAFKA-12407: Document Controller Health Metrics

### PR DESCRIPTION
**Pull request by dongjinleekr**
This PR fills the omitted documentation of:

- `kafka.controller:type=ControllerChannelManager,name=RequestRateAndQueueTimeMs,brokerId=([-.\w]+)`
- `kafka.controller:type=ControllerEventManager,name=EventQueueSize`
- `kafka.controller:type=ControllerEventManager,name=EventQueueTimeMs`

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by dongjinleekr** [comment]
@satishd @showuon Could you have a look? cc/ @lindong28

File: docs/ops.html [hunk 1]
```diff
@@ -1271,6 +1271,24 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
         <td>kafka.server:type=BrokerTopicMetrics,name=ReplicationBytesInPerSec</td>
         <td></td>
       </tr>
+      <tr>
+        <td>Controller Request rate from Broker</td>
+        <td>kafka.controller:type=ControllerChannelManager,name=RequestRateAndQueueTimeMs,brokerId=([-.\w]+)</td>
```
Comment (showuon) [inline docs/ops.html]: Our brokerId should be an Integer value, except I miss anything.
ref: http://kafka.apache.org/documentation/#brokerconfigs_broker.id

Comment (dongjinleekr) [reply docs/ops.html]: @showuon You are right. Fixed.

**Comment by lindong28** [comment]
Thanks you @dongjinleekr for the improvement! LGTM.

**Pull request merged**